cluster/gce/coreos/master.yaml

```diff
@@ -135,6 +135,7 @@ coreos:
         --reconcile-cidr=false
         Restart=always
         RestartSec=10
+        KillMode=process
```

Comment:
hm... what is the difference between kubelet on master node and on compute node?